style.css

```diff
@@ -1,8 +1,92 @@
 body {
-  box-sizing: border-box;
   margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.nav__bar {
+  background: #64b5f6;
+  color: white;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 2rem 0 2rem;
+}
+
+.auth__name {
+  text-align: left;
+  margin-left: 2rem;
+}
+
+.links {
+  margin-right: 3rem;
+
+  text-decoration: none;
+  color: white;
+  /*float: right;*/
+  align-items: center;
+
+
+}
 
+.nav-right {
+  text-align: right;
 }
-*, *:before, *:after {
-  box-sizing: inherit;
+
+.container {
+  /*grow up to 65rem*/
+  max-width: 70rem;
+  /*center if there is width left*/
+  /*margin: auto;*/
+  margin: 0 4rem  0  8rem;
+}
+
+.card {
+  /*width: calc(100% / 2 - 1rem - 10rem);*/
```

Comment:
Don't comment this out. Just make sure you subtract both the left AND right margin and this will work and make a 1/2 grid item!